.github/dependabot.yml

```diff
@@ -11,4 +11,22 @@ updates:
       interval: weekly
     groups:
       all-actions:
-        patterns: [ "*" ]
\ No newline at end of file
+        patterns: [ "*" ]
+  
+  # Maintain dependencies for Python packages
+  - package-ecosystem: pip
+    directory: /server
+    schedule:
+      interval: daily
+
+  # Maintain dependencies for Shell scripts
+  - package-ecosystem: shell
```

Comment:
I'm not sure `shell` is a supported `package-ecosystem`.  Do you have a link to a list of `package-ecosystems` that indicates `shell` is supported?